README.md

```diff
@@ -123,7 +148,19 @@ For a complete list of our pods, check out the .podspec files in the root direct
 
 ### Frameworks
 
-1. Download the [latest SDK](https://sdk-for-ios.amazonwebservices.com/latest/aws-ios-sdk.zip). Older SDK versions can be downloaded from `https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-#.#.#.zip`, where `#.#.#` represents the version number. So for version 2.10.2, the download link is [https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-2.10.2.zip](https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-2.10.2.zip).
+#### XCFramework setup
+
+Starting AWS SDK iOS version 2.22.1, frameworks are released as XCFramework. Following the steps below to instal XCFramework.
+
+1. Download the [latest SDK](https://sdk-for-ios.amazonwebservices.com/latest/aws-ios-sdk.zip). Older SDK versions can be downloaded from `https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-#.#.#.zip`, where `#.#.#` represents the version number. So for version 2.22.1, the download link is [https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-2.22.1.zip](https://sdk-for-ios.amazonwebservices.com/aws-ios-sdk-2.22.1.zip).
+> Note: If you are using version < 2.22.1 please refer to the section below where frameworks are added.
+
+4. On your application targets’ General settings tab, in the Embedded Binaries section, drag and drop each xcframework you want to use from the downloaded folder.
```

Comment:
```suggestion
2. Uncompress the ZIP file
3. On your application targets’ General settings tab, in the Embedded Binaries section, drag and drop each xcframework you want to use from the downloaded folder.
```


Also, will individual SDK ZIPs come when we do SPM support or is that something we need to document now?

Reply:
It is part of SPM, we can add it later.